.github/workflows/scripts/e2e-utils.sh

```diff
@@ -93,7 +93,7 @@ e2e_verify_predicate_v1_buildDefinition_buildType() {
 }
 
 e2e_verify_predicate_v1_buildDefinition_systemParameters() {
-    _e2e_verify_query "$1" "$3" '.buildDefinition.systemParamters.'"$2"
+    _e2e_verify_query "$1" "$3" '.buildDefinition.systemParamters."$2".value'
```

Comment:
> Expressions don't expand in single quotes, use double quotes for that.

Yeah, this needs to be fixed. Also parameters was misspelled.
```suggestion
    _e2e_verify_query "$1" "$3" ".buildDefinition.systemParameters.\"$2\".value"
```

Reply:
thank you!